CHANGELOG.md

```diff
@@ -3,6 +3,106 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 7.2.0
+
+_Apr 4, 2024_
+
+We'd like to offer a big thanks to the 18 contributors who made this release possible. Here are some highlights ✨:
+
+- 🎁 Allow higher Data Grid packages' props to be used in MIT (#12365) @MBilalShafi
```

Comment:
I wouldn't put this into highlights, as it's an internal change and has no impact on the users